README.md

```diff
@@ -163,12 +199,12 @@ The variable `XDG_CONFIG_HOME` is defined by the
 as the location of user specific configuration.
 If not set or empty, it has the following system-dependent defaults:
 
-* Unix: `$HOME/.config`
-* Windows: `%APPDATA%` (`C:\Users\<user>\AppData\Roaming`)
+- Unix: `$HOME/.config`
+- Windows: `%APPDATA%` (`C:\Users\<user>\AppData\Roaming`)
```

Comment:
Nor this.

Reply:
It's about consistency, something markdown linter complains about.